assets/src/hooks/v2/use_api_response.tsx

```diff
@@ -142,7 +142,10 @@ const useApiPath = (screenId: string, appendPath?: string): string => {
       last_refresh: getDatasetValue("lastRefresh"),
       requestor:
         getDatasetValue("requestor") ?? (isRealScreen() ? "real_screen" : null),
+      route_id: getDatasetValue("routeId"),
       screen_side: getScreenSide(),
+      stop_id: getDatasetValue("stopId"),
+      trip_id: getDatasetValue("tripId"),
```

Comment:
From huddle: wonder if we're overcomplicating this a bit by "ingesting" query params through the initial `ScreenController` page load, which then embeds them as data attributes, which are then picked up by the API data hook, which then sends them to the API. Considering having this hook directly copy query params from the current page URL, determined by optional prop passed down through `ScreenPage`/`ScreenContainer` which would only be used in the on-bus entrypoint.